index.html

```diff
@@ -0,0 +1,240 @@
+      <div class="presentation">
+        <h1>Walter Velasquez</h1>
+        <h2>Software Developer</h2>
+      </div>
+      
+      <div class="sup_text">
+        <p class="st">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don't hestiate to contact me.</p>
+        <img alt="divider vector" src="img/Vector.svg" class="divider_a">
+
+      <div class="sup_skills">
+        <ul class="sup_list">
+          <li class="list_title">Language</li>
+          <li class="sup_li">Javascript</li>
+          <li class="sup_li">Ruby</li>
+          <li class="sup_li">Html</li>
+          <li class="sup_li">Css</li>
+        </ul>
+        <ul class="sup_list">
+          <li class="list_title">Frameworks</li>
+          <li class="sup_li">Bootstrap</li>
+          <li class="sup_li">Ruby on Rails</li>
+          <li class="sup_li">RSPec</li>
+          <li class="sup_li">CapyBara</li>
+          <li class="sup_li">Selenium</li>
+        </ul>
+        <ul class="sup_list">
+          <li class="list_title">Skills</li>
+          <li class="sup_li">Codekit</li>
+          <li class="sup_li">Github</li>
+          <li class="sup_li">Codepen</li>
+          <li class="sup_li">Gitlab</li>
+          <li class="sup_li">Terminal</li>
+        </ul>
+      </div>
+      <img alt="divider vector" src="img/Vector.svg" class="divider_b">
+      </div>
+      <div class="sup_items">
+        <div class="work-btn l-btn">
+          <div class="arrow_d"><a href="#"><img alt="my work" src="img/arrow_d.svg"></a></div>
+          <div class="text"><a href="#">Work</a></div>
+        </div>
+        <ul class="social_media">
+          <li class="social_icon"><a href="https://github.com/Sirvelasque"><img alt="My Github" src="img/icons/icongit.svg"></a></li>
+          <li class="social_icon"><a href="https://www.linkedin.com/in/walter-velasquez-669633223/"><img alt="My in icon" src="img/icons/iconin.svg"></a></li>
+          <li class="social_icon"><a href="https://angel.co/profile/edit/overview"><img alt="My angelist" src="img/icons/iconangel.svg"></a></li>
+          <li class="social_icon"><a href="https://twitter.com/WalteWalter"><img alt="My twitter" src="img/icons/icontw.png"></a></li>
+          <li class="social_icon"><a href="https://www.google.com/"><img alt="My Medium" src="img/icons/iconm.png"></a></li>
+        </ul>
+        <div class="work-btn r-btn">
+          <div class="arrow_d"><a href="#"><img alt="my work" src="img/arrow_d.svg"></a></div>
+          <div class="text"><a href="#">Work</a></div>
+        </div>
+      </div>
+    </section>
+
+    <section class="works">
+      <div class="works_head">
+        <h2 class="work_title">MY RECENT WORK</h2>
+        <!--divider-->
+        <div class="divider_c"></div>
+      </div>
+
+      <div class="works_container">
+        <!-- reverse stands for the flex direction-->
```

Comment:
Please try to avoid using comments, because they make the code bulky, yet the simplier the code the easier to read.